### DISCUSSION TITLE
[Question]    What is Light vs Full  installer ???

### DISCUSSION THREAD

**Discussion by blackholeearth** [NickeManarin/ScreenToGif#1359] [Q&A] [answered]
**Question :**
 i dont know which one to install .
  my pc has latest win10.. but  my pc is  little slow .. from 2014.
  
  which one is right one for me??
  
 

**Describe the solution you'd like**
 i wish this info is written in the  readme.  near the download link.

**edit:**

| **Self Contained**                     | **Light**                                               |
| -------------------------------------- | ------------------------------------------------------- |
| .Net Runtime is Embedded into The App. | you have to Download and install .NET 8 Manually. |
|  -                                     |    [download - .NET 8](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-8.0.404-windows-x64-installer)                                                    |
| ScreenToGif.xxx.Portable -   75 MB     | ScreenToGif.xxx.Light.Portable   -   12.8 MB            |
| ScreenToGif.xxx.Setup   -   76.8 MB    | ScreenToGif.xxx.Light.Setup -   14.6 MB                 |

**Comment by NickeManarin** [comment, marked as answer]
Hello! The only difference is that the Light version doesnt have the .NET runtime alongside, so you need to download .NET and install separately.

**Reply by blackholeearth** [reply]
huumm thanks...
